src/zenml/integrations/kserve/steps/__init__.py

```diff
@@ -13,12 +13,8 @@
 #  permissions and limitations under the License.
 """Initialization for KServe steps."""
 
-
 from zenml.integrations.kserve.steps.kserve_deployer import (
     KServeDeployerStepConfig,
+    TorchServeParamters,
```

Comment:
```suggestion
    TorchServeParameters,
```

(This should also be renamed wherever it came from... This is the correct spelling).